pages/blog/hackathon-faq.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "AsyncAPI Hackathon FAQ"
+date: 2021-10-19T06:00:00+01:00
+type: Communication
+tags:
+  - Conference
+  - Hackathon
+cover: /img/posts/hackathon-faq/cover.webp
+authors:
+  - name: Lukasz Gornicki
+    photo: /img/avatars/lpgornicki.webp
+    link: https://twitter.com/derberq
+    byline: AsyncAPI Maintainer and Community Guardian
+excerpt: Answers to all the questions you ever wanted to ask about the AsyncAPI Hackathon 2021
+featured: true
+---
+
+Last year it was the first time we hosted the AsyncAPI Conference. This year it is time to host our first AsyncAPI Hackathon. Many questions are coming from the community, and this article tries to address them.
+
+### When is the Hackathon?
+
+1st - 31st of October 2021.
+
+You have an entire month to complete your idea. No usual over-weekend hard work but relaxed month instead.
+
+### What is the main theme for the Hackathon?
+
+The goal of the Hackathon is to provide MVP solutions that can help AsyncAPI Community in ways like:
+
+- Ease education and getting started with the project.
+- Bring new AsyncAPI-related solutions that can solve the existing challenges the community is facing with event-driven architectures.
+- Extend existing tools, reuse them in some scenarios, add the extra implementation on top.
+
+### Do I need to register for the event?
+
+No.
+
+Anybody can join the Hackathon. Whenever you want. No pressure in case you can't deliver what you planned.
+
+### Is it team competition or for individuals?
+
+Both.
+
+You can form a team if you want. Just keep in mind that once you win, the prize also goes to the entire team.
+
+### I want to participate, but I don't have a good idea
+
+No worries, look at ideas that [others shared](https://github.com/asyncapi/community/discussions/categories/asyncapi-hack-2021-ideas-brainstorming), you can pick one of them or just get some inspiration.
+
+### Can there be multiple submissions with the same idea?
+
+Yes.
+
+We do not set any limits here. The same idea can be done by multiple teams. Just keep in mind that if there are two excellent submissions for the same idea, I doubt judges will vote for both to let other ideas shine, although this is not an official rule.
```

Comment:
`"I doubt judges will vote for both to let other ideas shine, although this is not an official rule."`

Sorry, what? this confused me 😕

Reply:
I'm just trying to express that if we will have 5 people independently working on https://github.com/asyncapi/community/discussions/55 and event of 2-3 submissions will be super cool, I doubt that judges will choose all of them but most probably just one.

Reply:
Ok so are you saying that if a single person/team submits 2-3 ideas, judges will prob choose only 1? Because the sentence reads as if you were saying that if many people/teams submitted the same ideas, prob only one team will win. 

So I want to fix this sentence, and make sure it's clearer 😀

Reply:
oh, is that what it says 😄 

I'm saying that if multiple people/teams submit the same solution, but of course implemented differently,  then most probably judges will only select one and rest of votes will put on completely different solutions 😄

Reply:
okidoki, I guess it's ok then... sorry for all the back and forth questions, I was legit trying to make sure I read and understood it all correctly :D